pynest/nest/server/hl_api_server.py

```diff
@@ -382,16 +352,21 @@ def set_mpi_comm(comm):
 
 
 def run_mpi_app(host="127.0.0.1", port=5000):
-    # NEST segfaults if someone messes with the number of threads, so we don't.
+    # NEST segfaults if someone messes with the number of threads from
+    # the outside, so we don't.
     app.run(host=host, port=port, threaded=False)
 
 
 def combine(call_name, response):
     """Combine responses from different MPI processes.
 
-    This function combines the responses of all MPI processes and
-    returns a single response object. The type of the result can vary
-    depending on the call of the call that produced it.
+    If this function is run in a serial context, i.e., without MPI,
+    it returns the only response immediately.
+
+    In a distributed scenario, this function combines the responses of
+    all MPI processes and returns a single response object. The type
+    of the result can vary depending on the call of the call that
```

Comment:
This seems confusing to me "call of the call", could you clarify, maybe with an example?